test/performance/broker-latency/broker-latency-setup.yaml

```
@@ -35,6 +46,112 @@ spec:
 
 ---
 
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: eventing-broker-ingress
+  namespace: perf-eventing
+
+---
```

As discussed on Slack, filter Pods won't be created without that ServiceAccount.
```suggestion
---

apiVersion: v1
kind: ServiceAccount
metadata:
  name: eventing-broker-filter
  namespace: perf-eventing

---
```